torch/autograd/functional.py

```diff
@@ -565,32 +565,39 @@ def vhp(func, inputs, v=None, create_graph=False, strict=False):
         func (function): a Python function that takes Tensor inputs and returns
             a Tensor with a single element.
         inputs (tuple of Tensors or Tensor): inputs to the function ``func``.
-        v (tuple of Tensors or Tensor): The vector for which the vector Hessian product is computed. Must be the
-            same size as the input of ``func``. This argument is optional when
-            ``func``'s input contains a single element and (if it is not provided) will be set as a Tensor
-            containing a single ``1``.
-        create_graph (bool, optional): If ``True``, both the output and result will be
-            computed in a differentiable way. Note that when ``strict`` is ``False``, the result can not
-            require gradients or be disconnected from the inputs.
+        v (tuple of Tensors or Tensor): The vector for which the vector Hessian
+            product is computed. Must be the same size as the input of
+            ``func``. This argument is optional when ``func``'s input contains
+            a single element and (if it is not provided) will be set as a
+            Tensor containing a single ``1``.
+        create_graph (bool, optional): If ``True``, both the output and result
+            will be computed in a differentiable way. Note that when ``strict``
+            is ``False``, the result can not require gradients or be
+            disconnected from the inputs.
             Defaults to ``False``.
-        strict (bool, optional): If ``True``, an error will be raised when we detect that there exists an input
-            such that all the outputs are independent of it. If ``False``, we return a Tensor of zeros as the
+        strict (bool, optional): If ``True``, an error will be raised when we
+            detect that there exists an input such that all the outputs are
+            independent of it. If ``False``, we return a Tensor of zeros as the
             vhp for said inputs, which is the expected mathematical value.
             Defaults to ``False``.
 
     Returns:
-        func_output (tuple of Tensors or Tensor): output of ``func(inputs)``
-        vhp (tuple of Tensors or Tensor): result of the dot product with the same shape
-            as the inputs.
-    Example::
+        output (tuple): tuple with:
+            func_output (tuple of Tensors or Tensor): output of ``func(inputs)``
+
+            vhp (tuple of Tensors or Tensor): result of the dot product with the
+            same shape as the inputs.
+
```

Comment:
This section was reformatted to fit into 80 char lines, the real fix was the "Returns" formatting. The rendered text wants to use the first  line as a return type, so I had to play with it to get the formatting right.